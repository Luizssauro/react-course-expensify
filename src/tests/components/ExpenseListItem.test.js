import React from 'react';
import { shallow } from 'enzyme';
import ExpenseListItem from '../../components/ExpenseListItem';
import expenses from '../fixtures/expenses';

describe('<ExpenseListItem />', () => {
	test('should render ListItem from the first expense', () => {
		const wrapper = shallow(
			<ExpenseListItem key={expenses[0].id} {...expenses[0]} />
		);
		expect(wrapper).toMatchSnapshot();
	});
});
