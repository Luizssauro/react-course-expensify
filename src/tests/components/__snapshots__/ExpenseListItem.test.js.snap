// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ExpenseListItem /> should render ListItem from the first expense 1`] = `
<div>
  <h3>
     
    <Link
      to="/edit/1"
    >
      Gum
    </Link>
  </h3>
  <p>
    $1.95
    -
    December 31st, 1969
  </p>
</div>
`;
